cmp/grid/Grid.scss

```diff
@@ -289,4 +289,97 @@
     font-size: var(--xh-grid-tiny-header-font-size-px);
     padding: 0 var(--xh-grid-tiny-header-lr-pad-px);
   }
-}
\ No newline at end of file
+}
+
+
+//------------------------
+// Tree grid styling modes
+//------------------------
+
+.xh-grid--highlight-groups {
+
+  .ag-theme-balham.xh-ag-grid, .ag-theme-balham.xh-ag-grid--no-stripe-rows {
+    div[class*=" ag-row-group"], div[class^="ag-row-group"] {
+      &:not(.ag-row-selected), &.ag-row.ag-row-odd:not(.ag-full-width-row):not(.ag-row-selected) {
+        background-color: hsl(207, 90, 94);
```

Comment:
We should declare new CSS vars for this group background color and the border color.

Then it looks like we should be able to simply set them to different default values for the light/dark theme variations in `vars.scss` and not need to duplicate all these rules here.  That's the approach we should be using consistently for the grid and many other places.